pkg/cmd/rebuild.go

```diff
@@ -84,9 +108,10 @@ func (o *rebuildCmdOptions) listAllIntegrations(c client.Client) ([]v1.Integrati
 func (o *rebuildCmdOptions) getIntegrations(c client.Client, names []string) ([]v1.Integration, error) {
 	ints := make([]v1.Integration, 0, len(names))
 	for _, n := range names {
-		it := v1.NewIntegration(o.Namespace, n)
+		name := kubernetes.SanitizeName(n)
```

Comment:
Why do we need to sanitize the name? Isn't this value already the one passed by the user?